src/frontends/sql/ops/merge_op_test.cc

```diff
@@ -0,0 +1,130 @@
+//-----------------------------------------------------------------------------
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//----------------------------------------------------------------------------
+#include "src/frontends/sql/ops/merge_op.h"
+
+#include <algorithm>
+
+#include "src/common/testing/gtest.h"
+#include "src/common/utils/ranges.h"
+#include "src/frontends/sql/ops/example_value_test_helper.h"
+#include "src/frontends/sql/ops/sql_op.h"
+#include "src/ir/attributes/attribute.h"
+#include "src/ir/value.h"
+
+namespace raksha::frontends::sql {
+namespace {
+
+class MergeOpTest : public ::testing::TestWithParam<
+                        std::tuple<ir::ValueList, ir::ValueList>> {
+ public:
+  MergeOpTest() { SqlOp::RegisterOperator<MergeOp>(context_); }
+
+ protected:
+  ir::IRContext context_;
+};
+
+using ::testing::ElementsAreArray;
+using ::testing::Eq;
+using ::testing::Pair;
+using ::testing::UnorderedElementsAre;
```

Comment:
Should the `using` declarations be up at the top of the `namespace`?

Reply:
done.